src/NuGet.Services.EndToEnd/Support/Clients/GalleryClient.cs

```diff
@@ -156,6 +156,30 @@ public async Task UnlistAsync(string id, string version, ITestOutputHelper logge
             await SendToPackageAsync(HttpMethod.Delete, id, version, logger);
         }
 
+        public async Task SearchPackageODataV2FromDBAsync(string id, bool semver2, ITestOutputHelper logger)
+        {
+            var galleryEndpoint = await GetGalleryUrlAsync(logger);
+            var url = semver2 
+                ? $"{galleryEndpoint}/api/v2/Packages()?$filter=tolower(Id) eq '{id.ToLower()}'&$orderby=Id&semVerLevel=2.0.0"
```

Comment:
`eq` is case insensitive on non-hijacked queries.

Reply:
If we're trying to break hijacking, `and 1 eq 1` is an easy way

Reply:
Updated the query